ios/InstagramStories.m

```diff
@@ -135,9 +151,17 @@ - (void)shareSingle:(NSDictionary *)options
             } else {
                 UIImage *backgroundImage = [UIImage imageWithData: [NSData dataWithContentsOfURL:backgroundURL]];
                 UIImage *stickerImage = [UIImage imageWithData: [NSData dataWithContentsOfURL:sticketURL]];
-                
                 [self backgroundImage:UIImagePNGRepresentation(backgroundImage) stickerImage:UIImagePNGRepresentation(stickerImage) attributionURL:attrURL];
             }
+        } else if([method isEqualToString:@"shareBackgroundVideo"]) {
+            
+            NSString *URL = [RCTConvert NSString:options[@"backgroundVideo"]];
```

Comment:
One question: we need to pass a URL to the video or this will work with a base64?

Reply:
Need to pass URL. I'm not to sure about base64